src/LazyOperations/Complement.jl

```diff
@@ -36,6 +38,7 @@ struct Complement{N<:Real, S<:LazySet{N}}
 end
 
 isoperationtype(::Type{<:Complement}) = true
+isconvextype(::Type{<:Complement}) = false
```

Comment:
At some point (here or later) we should add that the `Complement` of `HalfSpace`, `EmptySet`, and `Universe` is convex (these are the only cases I can think of).

Reply:
Good point. I've opened https://github.com/JuliaReach/LazySets.jl/issues/1841 to follow up.